crypto/armcap.c

```diff
@@ -120,6 +145,9 @@ static unsigned long getauxval(unsigned long key)
 #  define HWCAP_CE_SHA256        (1 << 6)
 #  define HWCAP_CPUID            (1 << 11)
 #  define HWCAP_CE_SHA512        (1 << 21)
+                                  /* AT_HWCAP2 */
+#  define HWCAP2                 26
+#  define HWCAP2_RNG		 (1 << 16)
```

Comment:
Make things line up?  Not essential but it is easier to read.

Reply:
fixed